readme.md

```diff
@@ -246,6 +246,20 @@ $ git commit --amend
 $ git reset HEAD~
 ```
 
+>Get a list of branches ordered by most recent commit
```

Comment:
Just add header above this new changes to separate.

Header Title: Show Sorted Branch

Reply:
Done :fire: